CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 - Do not report SIC_INNER_SHOULD_BE_STATIC for classes annotated with JUnit Nested ([#560](https://github.com/spotbugs/spotbugs/issues/560))
 - Detect created, but not-thrown exceptions, which are created by not the constructor ([#2547](https://github.com/spotbugs/spotbugs/issues/2547))
 - Fixed eclipse plugin Effort.values pass to effortViewer as required cast to varargs ([#2579](https://github.com/spotbugs/spotbugs/pull/2579))
-
+- Fixed schema location for findbugsfilter.xsd ([[#1416](https://github.com/spotbugs/spotbugs/issues/1416)])
```

Comment:
@NeetigyaPod This needs moved into new section at top as release just occurred, make sure you are rebased and you an create similar section as you used here in the unreleased area at top.

Reply:
Rectified.